src/test/java/it/mulders/mcs/search/FormatTypeTest.java

```diff
@@ -0,0 +1,55 @@
+package it.mulders.mcs.search;
+
+import it.mulders.mcs.search.printer.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.*;
```

Comment:
Could you please not use wildcard imports here?

Reply:
Wildcard imports replaced.